app/policies/user_policy.rb

```diff
@@ -11,8 +11,12 @@ def create?
     is_admin? || is_himself?
```

Comment:
Don't know if this makes sense, the policy `create?` with a check if `is_himself?`

I believe that this is some kind of bug, introduced on this commit: https://github.com/Codeminer42/cm42-central/commit/78d577a9682bc4c519c1fba8570d4af487cd3ae9
It made sense to have the `is_himself?` check when updating, but when changed to `create?` it lost its purpose. 
And even, introduced the bug that this PR is trying to solve.